runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -511,11 +509,12 @@ private void cleanUpForRetry() {
         runtime.getAddressSpaceView().invalidateClientCache();
         runtime.getObjectsView().getObjectCache().clear();
         runtime.getStreamsView().getStreamCache().clear();
-        nextRead = logHead;
-        resetAddressProcessed();
 
         // Re ask for the Head, if it changes while we were trying
         findAndSetLogHead();
+
```

Comment:
Can you please add a comment, so in the future it is clear that this ensures that the loading process continues from the trim mark.

Reply:
done